tests/e2e/system_test/test_reclaim_space_cronjob.py

```diff
@@ -0,0 +1,211 @@
+from ocs_ci.helpers.performance_lib import run_oc_command
+from ocs_ci.helpers import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import PVCNotCreated
+
+logger = logging.getLogger(__name__)
+ERRMSG = "Error in command"
+
+
+@tier3
+@skipif_ocp_version("<4.14")
+@skipif_ocs_version("<4.14")
+class TestReclaimSpaceCronJob(ManageTest):
+    """
+    Test that TODO
+    """
+
+    def setup(self):
+        self.pvc_objs_created = []
+        self.temp_files_list = []
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        """
+        Cleanup the test environment
+
+        """
+
+        def finalizer():
+            # pvc objects usually are deleted in the test (and the second delete has no effect),
+            # if for some reason the pvcs were not deleted in the test - they will be deleted here
+            for pvc_obj in self.pvc_objs_created:
+                pvc_obj.delete()
+
+            for temp_file in self.temp_files_list:
+                if os.path.exists(temp_file):
+                    run_cmd(f"rm {temp_file}")
+
+            run_oc_command(cmd=f"delete namespace {self.namespace}")
+
+        request.addfinalizer(finalizer)
+
+    def test_reclaim_space_cronjob(self):
+        """
+        Test case to check reclaim space cronjobs are created correctly for rbd pvcs in openshift-* namespace
+        """
+        num_of_samples = 10
+        namespace = f"openshift-{uuid4().hex}"
+        self.namespace = namespace
+        res = run_oc_command(cmd=f"create namespace {self.namespace}")
+        assert ERRMSG not in res[0], (
+            f"Failed to create namespace with name {namespace}" f"got result: {res}"
+        )
+        logger.info(f"Namespace {namespace} created")
+
+        res = run_oc_command(cmd=f"get namespace {namespace} -o yaml")
+        namespace_dict = yaml.safe_load("\n".join(res))
+
+        schedule = namespace_dict["metadata"]["annotations"][
+            "reclaimspace.csiaddons.openshift.io/schedule"
+        ]
+        assert (
+            schedule == "@weekly"
+        ), f"Namespace {namespace} created with schedule {schedule}, expected @weekly"
+        logger.info(f"Existence of schedule {schedule} validated.")
+
+        for i in range(num_of_samples):
+            try:
+                pvc_obj = helpers.create_pvc(
+                    sc_name=constants.DEFAULT_STORAGECLASS_RBD,
+                    size="1Gi",
+                    namespace=namespace,
+                )
+                helpers.wait_for_resource_state(pvc_obj, constants.STATUS_BOUND)
+            except Exception as e:
+                logger.exception(f"The PVC was not created, exception [{str(e)}]")
+                raise PVCNotCreated("PVC did not reach BOUND state.")
+
+            logger.info(f"PVC by name {pvc_obj.name} created")
+            self.pvc_objs_created.append(pvc_obj)
+
+        # wait since it may take some time for cronjobs to be created
+        time.sleep(30)
```

Comment:
i would suggest to declare 30 as const and use it in multiple places

Reply:
Fixed here and in other places. Thank you!

Reply:
time.sleep(30) can be changed here to time.sleep(timeout)

Reply:
Fixed, thank you!